src/View/Helper/FormElementErrors.php

```diff
@@ -65,22 +65,23 @@ public function __invoke(ElementInterface $element = null, array $attributes = [
     public function render(ElementInterface $element, array $attributes = [])
     {
         $messages = $element->getMessages();
-        if (empty($messages)) {
-            return '';
-        }
-
-        $messages = $messages instanceof Traversable ? iterator_to_array($messages) : $messages;
-        if (! is_array($messages)) {
+        if ($messages instanceof Traversable) {
+            $messages = iterator_to_array($messages);
```

Comment:
Also use statement.
```suggestion
            $messages = ArrayUtils::iteratorToArray($messages);
```

Reply:
Similar situation as above, there is no reason to change it, unless you can provide a valid test case. Please note that I kept the same code as before, just optimised the condition.